loky/process_executor.py

```diff
@@ -374,15 +394,18 @@ def _process_worker(call_queue, result_queue, initializer, initargs,
     # set the global _CURRENT_DEPTH mechanism to limit recursive call
     global _CURRENT_DEPTH
     _CURRENT_DEPTH = current_depth
+    _REFERENCE_PROCESS_SIZE = None
+    _LAST_MEMORY_CHECK = None
```

Comment:
Not sure why it is better in upper case?
Reading the code, I though it was global variable so I would use lower case here but this is just nitpicking.

Reply:
done